test/util.js

```diff
@@ -0,0 +1,14 @@
+require('ava')
```

Comment:
Is this ava requirement needed?

Reply:
`nyc ava` will complain if it's not. I couldn't find a way to exclude the file entirely from nyc